parent/pom.xml

```diff
@@ -429,6 +472,25 @@
                                         <ignore></ignore>
                                     </action>
                                 </pluginExecution>
+                                <pluginExecution>
+                                    <pluginExecutionFilter>
+                                        <groupId>
```

Comment:
Please correct (unify) XML formatting.

Reply:
`<groupId>org.codehaus.mojo</groupId>` and so on.

Reply:
OK. Can you please provide the XML formatter settings so that we can document this?